### ISSUE TITLE
format: "textarea" - looks bad

### ISSUE THREAD

**Issue by DriveSoft** [premieroctet/next-admin#142]
Hi.

Can you check textarea in form, something wrong with styles.

example:
```
      ModelName: {
         edit: {
            display: ["name", "info"],
            fields: {
               info: {
                  format: "textarea",
               }
            } 
         }
      },
```

![image](https://github.com/premieroctet/next-admin/assets/11293522/bbc0a6a5-1d1e-4bbe-bba5-e57f9c941035)

**Comment by cregourd** [comment]
In roadmap, will be fixed soon

**Comment by cregourd** [comment]
This is fixed on [v3.2.6](https://www.npmjs.com/package/@premieroctet/next-admin/v/3.2.6) 🚀
Feel free to give feedback
I close the issue
